libs/langchain-ollama/src/utils.ts

```diff
@@ -14,6 +14,9 @@ import type {
 } from "ollama";
 import { v4 as uuidv4 } from "uuid";
 
+// Track previous thinking content to calculate incremental changes
+let previousThinkingContent = "";
```

Comment:
I am not sure if we should use a global environment like this to store thinking tokens. What happens if someone works with multiple chat models and invokes them at the same time?